uportal-war/src/main/java/org/jasig/portal/groups/pags/testers/PropertyInvertedRegexTester.java

```diff
@@ -37,10 +38,25 @@
 
     private Logger logger = LoggerFactory.getLogger(this.getClass());
 
+    public PropertyInvertedRegexTester(IPersonAttributesGroupTestDefinition definition) {
+        super(definition);
+        final String propertyName = definition.getTestValue();
+        String regexExpression = PropertiesManager.getProperty
+                (propertyName, "");
+        if (StringUtils.isBlank(regexExpression)) {
+            logger.error("Unable to find property name {} in portal.properties or has empty value."
+                    + " PAGS PropertyInvertedRegexTester will always return true for attribute {}",
+                    propertyName, definition.getAttributeName());
+        }
+        setPattern(regexExpression);
+    }
```

Comment:
Minor, but rather than duplicating lines 44 - 51, perhaps put them in a private method invoked by both constructors.